### PR TITLE
Release/alpha 0.0.1

### PR DESCRIPTION
**Pull request by MariyaSin**

### CODE REVIEW

**Review by VLiamin — APPROVED** [review]

**Review by Cheephot — APPROVED** [review]

**Review by borovHokage — APPROVED** [review]

File: src/TimeService.Business/Commands/Import/ImportStatCommand.cs [hunk 1]
```diff
@@ -380,18 +380,22 @@ private void AddHeaderCell(IXLWorksheet ws, int column, string value, XLColor co
...
-          List<DbLeaveTime> usersLeaveTimes = leaveTimes.Where(lt => lt.UserId == usersInfos[number].Id).ToList();
-
-          for (int i = 0; i < leaveTypeCount; i++)
+          for (int number = 0; number < usersInfos.Count; number++)
           {
-            ws.Cell(row, column + i).SetValue(
-                usersLeaveTimes
-                .Where(lt => lt.LeaveType == i)
-                .Select(lt => GetLeaveTimeOurs(monthsLimits, lt, filter.Year, filter.Month)).Sum());
+            int row = number + 3;
```
Comment (Glunky) [inline src/TimeService.Business/Commands/Import/ImportStatCommand.cs:387]: magic number - need constant

File: src/TimeService.Business/Commands/Import/ImportStatCommand.cs [hunk 2]
```diff
@@ -546,11 +550,11 @@ public OperationResultResponse<byte[]> Execute(ImportStatFilter filter)
 
       if (filter.DepartmentId.HasValue)
       {
-        usersIds = FindDepartmentUsers(filter.DepartmentId.Value, errors);
+        usersIds = FindDepartmentUsers(filter.DepartmentId.Value, new DateTime(filter.Year, filter.Month, 1), errors);
 
         projectsUsers = GetProjectsUsers(usersIds, errors);
 
-        projects = GetProjects(projectsUsers.Select(pu => pu.ProjectId).Distinct().ToList(), false, errors);
+        projects = GetProjects(projectsUsers?.Select(pu => pu.ProjectId).Distinct().ToList(), false, errors);
```
Comment (Glunky) [inline src/TimeService.Business/Commands/Import/ImportStatCommand.cs:557]: if projectsUsers is null - will be crash

**Review by Glunky — APPROVED** [review]

File: src/TimeService.Business/Commands/Import/ImportStatCommand.cs (hunk 2 above)
Comment (MariyaSin) [reply src/TimeService.Business/Commands/Import/ImportStatCommand.cs:557]: created task 889

File: src/TimeService.Business/Commands/Import/ImportStatCommand.cs (hunk 1 above)
Comment (MariyaSin) [reply src/TimeService.Business/Commands/Import/ImportStatCommand.cs:387]: created task 889

**Pull request merged**